vnext/Microsoft.ReactNative/ReactApplication.idl

```diff
@@ -5,29 +5,42 @@ import "ReactNativeHost.idl";
 #include "NamespaceRedirect.h"
 #include "DocString.h"
 
-namespace Microsoft.ReactNative {
-
+namespace Microsoft.ReactNative
+{
   [webhosthidden]
   [default_interface]
-  DOC_STRING("`ReactApplication` provides a base application class for use in applications that are entirely written in React Native. `ReactApplication` will load the React instance on launch of your app for you and provide accessors to your application's @ReactInstanceSettings and @ReactNativeHost to customize your React instance.")
-  unsealed runtimeclass ReactApplication : XAML_NAMESPACE.Application {
+  DOC_STRING(
+    "`ReactApplication` provides a base application class for use in applications that are entirely written "
+    "in React Native. `ReactApplication` will load the React instance on launch of your app for you and provide "
```

Comment:
```suggestion
    "in React Native. When the app launches, `ReactApplication` will load the React instance and provide "
```

Reply:
It looks to complex and repeats the loading conditions.